rules/falco_rules.yaml

```diff
@@ -2210,6 +2210,26 @@
   priority: NOTICE
   tags: [network, aws, container, mitre_discovery]
 
+
+# This rule is not enabled by default, since this rule is for GCP only.
+# If you want to enable this rule, overwrite the first macro,
+# And you can filter the container that you want to allow access to metadata by overwriting the second macro.
+- macro: consider_gce_metadata_access
+  condition: (never_true)
+
+- macro: gce_metadata_containers
+  condition: (k8s.ns.name = "kube-system")
+
+# On GCE instances, 169.254.169.254 is a special IP used to fetch
+# metadata about the instance. The metadata could be used to get credentials by attackers.
+- rule: Contact GCE Instance Metadata Service From Container
```

Comment:
This rule looks similar to rule `Contact EC2 Instance Metadata Service From Container`. Can you make this rule a more generic one by removing `GCE`? With the conditions we may want to have two macros:
1. `consider_metadata_access`
2. `user_known_metadata_access`

Reply:
Hi, @Kaizhe 
Thank you for your review.
I didn't know Azure's Metadata's IP Address. but it seems Azure uses same IP Address (169.254.169.254) for metadata service. so I think the rule can be more generic too.
Ref: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service

I will remove `GCE` as you suggest. Thank you.

Reply:
@Kaizhe I renamed the macros and rule names.
Could you review them again?

Reply:
The changes look great! Thank you for the contribution!

Reply:
Thank you for your review. I'm glad to contribute Falco.